docs/glossary.asciidoc

```diff
@@ -0,0 +1,32 @@
+[id="{p}-glossary"]
+== Glossary
+
+This glossary covers Elastic Cloud on Kubernetes. If a term is not in this list, you can also look in the link:https://kubernetes.io/docs/reference/glossary/[Kubernetes glossary].
+
+[[GKE]]GKE::
+  Google Kubernetes Engine (GKE): is a managed, production-ready environment for deploying containerized applications.
```

Comment:
Should the wording for all providers be similar?

Reply:
++ to consistent wording for all providers. Let's link to their respective product pages and only expand the acronym

GKE - https://cloud.google.com/kubernetes-engine/
AKS - https://azure.microsoft.com/en-us/services/kubernetes-service/
EKS - https://aws.amazon.com/eks/
OpenShift - https://www.openshift.com/
Kubernetes - https://kubernetes.io/